### PR TITLE
release: 2024-10-21

### PR DESCRIPTION
**Pull request by chaitanyabhandari**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,48 @@
 Subscribe to the [tracking issue #2231](https://github.com/tigerbeetle/tigerbeetle/issues/2231)
 to receive notifications about breaking changes!
 
+## TigerBeetle 0.16.10
+
+Released: 2024-10-21
+
+
+### Safety And Performance
```
Comment (matklad) [inline CHANGELOG.md]:
```suggestion
Released: 2024-10-21

### Safety And Performance
```

Extra blank here!

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,48 @@
...
+
+- [#2414](https://github.com/tigerbeetle/tigerbeetle/pull/2414)
+
+Ensure a replica only repairs the portion of the WAL that is required to step up as primary, instead
+of repairing the WAL in its entirety. This improves performance & availability during view change.
```
Comment (matklad) [inline CHANGELOG.md]: Could you also indent this a bit to match the other changelogs?

File: CHANGELOG.md (hunk 1 above)
Comment (chaitanyabhandari) [reply CHANGELOG.md]: Good catch, fixed!

File: CHANGELOG.md (hunk 2 above)
Comment (matklad) [reply CHANGELOG.md]: Stylistically, given that the reader might not have full context, I'd swap this around:

Improve performance and availability during view change by ...

Comment (chaitanyabhandari) [reply CHANGELOG.md]: Fixed both in https://github.com/tigerbeetle/tigerbeetle/commit/c154379ddb7f01d2c0101af6299ebae61ea7a5fc!

**Review by matklad — APPROVED** [review]

**Pull request merged**